### DISCUSSION TITLE
[Feature] Automatically create albums based on folders in external library

### DISCUSSION THREAD

**Discussion by take31415** [immich-app/immich#8596] [Feature Request]
### The feature

I'm a fan of the "External library" feature. But currently it's only importing its contents without any structure.
I'd like to have an album created per folder within the external library path. Currently additional manual effort is needed to create albums all the time for new pictures.

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
Duplicate of #4279
